justbuild.it-ConsoleApp/src/main/java/com/isa/control/SubMenuNavigator.java

```diff
@@ -0,0 +1,38 @@
+package com.isa.control;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public abstract class SubMenuNavigator {
+
+    void choose(){
```

Comment:
samo choose to za mało. Metoda powinna mieć nazwę czynności i co robi. Może Jakieś subMenuActions?

Reply:
poprawione